knimin/templates/pm_plate_list.html

```diff
@@ -0,0 +1,173 @@
+<!-- The Plate Mapper module -->
+<!-- Plate list -->
+
+{% extends logged_in_index.html %}
+{% block head %}
+{% from json import dumps %}
+<script type="text/javascript" src="/static/js/labmin.js"></script>
+<script type="text/javascript" src="/static/vendor/js/jquery.validate.min.js"></script>
+<script type="text/javascript" src="/static/vendor/js/jquery.dataTables.min.js"></script>
+<link rel="stylesheet" type="text/css" href="/static/vendor/css/jquery.dataTables.min.css">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.6.3/css/font-awesome.min.css">
```

Comment:
Just for future reference, the fact that is open source doesn't necessarily mean that we can use. If the license of FontAwesome is GPL, we will need to change the license of our code to be GPL, because our code is BSD and GPL is a viral license. However, it looks like you are using the CSS from FontAwesome and they've a MIT license, which I think it is fine to use. The licensing world is complex and we had some issues in the past, and it is really important to fulfill the licenses. @ElDeveloper I think we will need to add the FontAwesome license to our repository to be compliant with it? Which strategy did you follow in Emperor?